<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Сортировка таблицы</string>
  </property>
  <widget class="QTableWidget" name="mainTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>521</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="randomFill">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>30</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Заполнить
 случайными
 значениями</string>
   </property>
   <property name="iconSize">
    <size>
     <width>12</width>
     <height>12</height>
    </size>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="typeOfSorting">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>100</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Пузырьком</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Быстрая</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Расчёска</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Гномья</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Обезьянья</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="sortButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>150</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Сортировка</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="numOfColumns">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Кол-во столбцов</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="numOfRows">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Кол-во строк</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="maxInt">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Макс. случ. знач.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="minInt">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Мин. случ. знач.</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textOut">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>200</y>
     <width>281</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="visualCheck">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>90</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Визуализация  
(фича тестируется)</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>30</y>
     <width>302</width>
     <height>35</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="getMinButton">
      <property name="text">
       <string>Минимум</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="getMaxButton">
      <property name="text">
       <string>Максимум</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="meanInt">
      <property name="text">
       <string>Ср.знач.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
